### DISCUSSION TITLE
Change in R² following the modification of the Adonis function to Adonis2 ?

### DISCUSSION THREAD

**Discussion by LoryA972** [vegandevs/vegan#519] [Q&A]
Hi !

I have a question about the function Adonis2:

The R² results obtained with the "Adonis" function have changed after re-running the dataset with the new "Adonis2" function in Vegan. Does anyone know why or have had a similar problem?

Thank for your answer.

**Comment by jarioksa** [comment]
No, nobody knows. A bit more information would be useful. The output would be the minimum needed.

**Reply by LoryA972** [reply]
Thank you Jari for your answer.

I would like to know the contribution of several environmental factors on the community structure.
I have a distance matrix calculated with jaccard that I used to calculate the percentage contribution of each factor.

permanovaMM.jacc1_loc=adonis(matrix_jacc1 ~ Location, data = env, permutations=999)

A few months ago, I did the analysis when the adonis function was available and got results.

In the meantime, vegan has been updated and the "adonis" function has become obsolete and has been replaced by "adonis2". I ran the same dataset and I get different results between "adonis" and "adonis2"...

permanovaMM.jacc1_loc=adonis2(matrix_jacc1 ~ Location, data = env, permutations=999)

Thank you for your help.

Lory-Anne

**Reply by gavinsimpson** [reply]
You've got to help us to help you! What does "different results" mean? And please don't describe the differences, *show* us so we can see exactly what output you see.

**Comment by LoryA972** [comment]
Thank you Jari for your answer. 

I would like to know the contribution of several environmental factors on the community structure. 
I have a distance matrix calculated with jaccard that I used to calculate the percentage contribution of each factor. 

permanovaMM.jacc1_loc=adonis(matrix_jacc1 ~ Location, data = env, permutations=999) 

A few months ago, I did the analysis when the adonis function was available and got results. 

In the meantime, vegan has been updated and the "adonis" function has become obsolete and has been replaced by "adonis2". I ran the same dataset and I get different results between "adonis" and "adonis2"... 

permanovaMM.jacc1_loc=adonis2(matrix_jacc1 ~ Location, data = env, permutations=999) 

Thank you for your help. 

Lory-Anne 

De: "Jari Oksanen" ***@***.***> 
À: "vegandevs/vegan" ***@***.***> 
Cc: "LoryA972" ***@***.***>, "Author" ***@***.***> 
Envoyé: Vendredi 10 Juin 2022 15:30:16 
Objet: Re: [vegandevs/vegan] Change in R² following the modification of the Adonis function to Adonis2 ? (Discussion #519) 





No, nobody knows. A bit more information would be useful. 

— 
Reply to this email directly, [ https://github.com/vegandevs/vegan/discussions/519#discussioncomment-2923066 | view it on GitHub ] , or [ https://github.com/notifications/unsubscribe-auth/AZSHVXTLSGSTH2EIVTQA5YTVOM7ORANCNFSM5YNAX3LA | unsubscribe ] . 
You are receiving this because you authored the thread. Message ID: <vegandevs/vegan/repo-discussions/519/comments/2923066 @ github . com> 


-- 
Lory-Anne BAKER 
Doctorante en Radioécologie 
LMGE-UMR 6023 & GEOLAB-UMR 6042 
Université de Clermont Auvergne 
Mél. [ ***@***.*** | ***@***.*** ] 
***@***.***
